packages/e2e/features/ui/components/storage/storage-image/load-private-image-from-s3.feature

```diff
@@ -0,0 +1,15 @@
+Feature: Load an image from S3 with private access level settings
+
+  Background:
+    Given I'm running the example "ui/components/storage/storage-image/private-access-level"
+
+  @react
+  Scenario: I successfully load a private image
+    When I type my "email" with status "CONFIRMED"
+    And I type my password
+    And I click the "Sign in" button
+    Then I see "The private image is loading."
+    Then I see "The private image is loaded."
```

Comment:
If I understand correctly, this only tests that the onLoad callback executed correctly, not that the image actually is displayed. Can we add a test to check for the image? Retrieve by test ID ideally or even by alt (I saw it defined)

```
cy.get('[alt="This is a private image."]')
.should('be.visible')
```
Same comment for protected/public scope.

Reply:
Think a broken image will also be rendered on page with all the props?